README.md

```diff
@@ -41,3 +43,7 @@ The primary contact for this project is [@jackrobison](mailto:jack@lbry.io).
 ## Additional information and links
 
 The documentation for the API can be found [here](https://lbry.tech/api/sdk).
+
+Daemon ports, settings, and configurations are documented [here](https://lbry.tech/resources/daemon-settings).
+
+An example configuration file can be found [here](https://github.com/lbryio/lbry/blob/master/lbrynet/conf.py).
```

Comment:
Could you add an example `daemon_settings.yml` file? This links to conf.py.

Reply:
@jessopb I added an example config here: https://github.com/lbryio/lbry/blob/master/example_daemon_settings.yml

Reply:
Done. Final?